pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/reader/JacksonJsonReader.java

```diff
@@ -27,11 +27,11 @@
 import java.io.IOException;
 import java.io.InputStream;
 
-public class JsonReader<T> implements SchemaReader<T> {
+public class JacksonJsonReader<T> implements SchemaReader<T> {
```

Comment:
Can we be sure nobody is using this class (since renaming is a backward-breaking change)? (Same question for the writer.)

Reply:
Well .. yeah someone could have used this class, for whatever reason from outside the framework. But it's original name is bad chosen, probably keep the old class and make it deprecated as well as the renamed one?

Reply:
The rename is not worth it, IMO, but I defer to maintainers.

Reply:
I kept both versions and deprecated the old ones.